Packages/com.unity.inputsystem/InputSystem/Actions/InputAction.cs

```diff
@@ -188,6 +188,22 @@ public ReadOnlyArray<InputControl> controls
             }
         }
 
+        public bool initialStateCheck
+        {
+            get => (m_Flags & ActionFlags.InitialStateCheck) != 0;
+            set
+            {
+                if (enabled)
+                    throw new InvalidOperationException(
+                        $"Cannot change the 'initialStateCheck' flag of action '{this} while the action is enabled");
```

Comment:
Why not?

Reply:
We'd have to sync `InputActionState` somehow. Either with a separate codepath or by re-resolving. I'm thinking it's not worth the trouble. In general, the action APIs make the assumption that configuration APIs require actions to be disabled.